qcfractal/server.py

```diff
@@ -268,7 +268,7 @@ def __init__(
             "heartbeat_frequency": self.heartbeat_frequency,
             "version": get_information("version"),
             "query_limit": self.storage.get_limit(1.0e9),
-            "client_lower_version_limit": "0.12.0",  # Must be XX.YY.ZZ
+            "client_lower_version_limit": "0.12.1",  # Must be XX.YY.ZZ
```

Comment:
Was this supposed to have happened during the 0.12.1 release? Just because I expected this to be 0.12.2, but also I don't think 0.12.2 has any client-server compatibility breaking changes.

Reply:
Usually we do not need to do this, but there is a bug that slipped through REST API changes.